### DISCUSSION TITLE
Multiple email addresses

### DISCUSSION THREAD

**Discussion by olivercoad** [thinkst/canarytokens#250] [Feature Requests]
It would be nice to be able to provide multiple email addresses for notifying multiple people. It would be reasonable to limit it to a very small number of comma separated emails.

**Comment by JStyle21** [comment]
I'm also interested in this

**Comment by JStyle21** [comment]
As a workaround i've set forwarding rules in gmail to send it to multiple accounts but it should be added to the codebase as the email apis allow this.
